ext_emconf.php

```diff
@@ -8,10 +8,10 @@
     'constraints' => [
         'depends' => [
             'php' => '7.4.0-8.1.99',
-            'typo3' => '10.4.11-11.5.99',
-            'extbase' => '10.4.11-11.5.99',
-            'fluid' => '10.4.11-11.5.99',
-            'frontend' => '10.4.11-11.5.99',
+            'typo3' => '11.5.0-11.5.99',
```

Comment:
To be consistent with the requirements in the `composer.json`, this needs to be `11.5.2-…`.